homeassistant/components/lovelace/__init__.py

```diff
@@ -86,11 +86,13 @@ async def reload_resources_service_handler(service_call: ServiceCall) -> None:
 
         config = await async_process_component_config(hass, conf, integration)
 
+        assert config is not None
```

Comment:
If the config has been reloaded the config validation can fail even though the integration config was valid when the integration was set up. So we probably need to check for None here and return or raise an error.